src/Common/src/Interop/Unix/libssl/Interop.libssl.msbuild

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <Compile Include="$(CommonPath)\Interop\Unix\libssl\Interop.libssl.cs">
+      <Link>Interop\Unix\libssl\Interop.libssl.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libssl\Interop.libssl_types.cs">
+      <Link>Interop\Unix\libssl\Interop.libssl_types.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libssl\Interop.OpenSsl.cs">
+      <Link>Interop\Unix\libssl\Interop.OpenSsl.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcrypto\Interop.X509.cs">
+      <Link>Interop\Unix\libcrypto\Interop.X509.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcrypto\Interop.X509Ext.cs">
+      <Link>Interop\Unix\libcrypto\Interop.X509Ext.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcrypto\Interop.BIO.cs">
+      <Link>Interop\Unix\libcrypto\Interop.BIO.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcrypto\Interop.ERR.cs">
+      <Link>Interop\Unix\libcrypto\Interop.ERR.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcrypto\Interop.ASN1.cs">
+      <Link>Interop\Unix\libcrypto\Interop.ASN1.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcrypto\Interop.Pkcs7.cs">
```

Comment:
```GetPkcs7Certificates``` in **Interop.Crypto.cs** needs ```SafePkcs7Handle``` which in turn needs ```Interop.Pkcs7.cs```

Interop.Crypto.cs is depended upon by  SafeX509Handles.Unix.cs